### DISCUSSION TITLE
Does `assistant-ui` support voice inputs?

### DISCUSSION THREAD

**Discussion by sunilswain** [assistant-ui/assistant-ui#1234] [Q&A] [answered]
Hey,

I am very much impressed by seeing the UI and its functionalities. I was wondering whether it is possible to add voice inputs?

**Comment by mariohmol** [comment]
Hi, curious to know if you have any news about it. Thanks

**Comment by AVGVSTVS96** [comment, marked as answer]
Hi, voice is coming soon!
